draft-ietf-rtcweb-jsep.xml

```diff
@@ -2667,21 +2669,26 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                     sections, the unmatched RtpTransceivers will need
                     to be associated in a subsequent offer.</t>
 
-                    <t>For each offered m= section, if the associated
-                    RtpTransceiver has been stopped, the corresponding
+                    <t>For each offered m= section, if any of the
+                    following conditions are true, the corresponding
                     m= section in the answer MUST be marked as rejected
                     by setting the port in the m= line to zero, as
                     indicated in <xref target="RFC3264"></xref>, Section
                     6., and further processing for this m= section can
-                    be skipped.</t>
+                    be skipped:
+                    <list style="symbols">
+                      <t>The associated RtpTransceiver has been stopped.
+                      </t>
 
-                    <t>For each offered m= section, if no supported codec
-                    is present in the offer, the corresponding
-                    m= section in the answer MUST be marked as rejected
-                    by setting the port in the m= line to zero, as
-                    indicated in <xref target="RFC3264"></xref>, Section
-                    6., and further processing for this m= section can
-                    be skipped.</t>
+                      <t>No supported codec is present in the offer.</t>
+
+                      <t>The bundle policy is "max-bundle" and the m=
+                      section is not in a BUNDLE group.</t>
+
+                      <t>The RTP/RTCP multiplexing policy is "require"
+                      and the m= section doesn't provide an
```

Comment:
For now I'm just restructuring the existing text here to make it more readable. I'll change "provide" to "contain" but we should probably do the generalizing it in a separate PR.